src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -1,17 +1,45 @@
 import React from 'react';
-import { focusElement } from 'platform/utilities/ui';
-import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
-import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
-import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
-import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+
+import { focusElement } from '@department-of-veterans-affairs/platform-utilities/ui';
+import FormTitle from '@department-of-veterans-affairs/platform-forms-system/FormTitle';
+// eslint-disable-next-line @department-of-veterans-affairs/use-workspace-imports
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
```

Comment:
Unfortunately I had to disable the linter warning here.  Making the change which satisfied the linter caused the import to break during the cypress tests.  I dunno.